src/basho_bench.erl

```diff
@@ -21,13 +21,18 @@
 %% -------------------------------------------------------------------
 -module(basho_bench).
 
--export([main/1, md5/1]).
+-export([run_benchmark/0, main/1, md5/1]).
 -include("basho_bench.hrl").
 
 %% ====================================================================
 %% API
 %% ====================================================================
 
+run_benchmark() ->
+    application:set_env(basho_bench_app, is_running, true),
+    basho_bench_sup:start_child(),
+    ok = basho_bench_worker:run(basho_bench_worker_sup:workers()).
+
```

Comment:
This function seems a bit odd given that it's always run from main/1. Seems like we're missing a way to initialize a benchmark config manually without actually running it.